### PR TITLE
remove pallet::getter usage from pallet-timestamp

### PR DESCRIPTION
**Pull request by tesol2y090**
this pr is a part of #3326

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 2/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5245611

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5245593

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 1/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5245610

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 3/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5245612

**Review by ggwpez — APPROVED** [review]

**Review by gupnik — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5247147

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 3/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5247166

**Review by kianenigma — APPROVED** [review]

**Review by franciscoaguirre — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5255029

**Comment by muraca** [comment]
CI is failing as there are various references to the `Timestamp::now()` function in other pallets (and I believe runtimes as well).
I think this is one of the few cases where an explicit getter should be preferred.

**Comment by tesol2y090** [comment]
> CI is failing as there are various references to the `Timestamp::now()` function in other pallets (and I believe runtimes as well). I think this is one of the few cases where an explicit getter should be preferred.

Yeah, they are in runtime and other pallets. I'll change it.

**Comment by muraca** [comment]
@tesol2y090 thanks! We need a prdoc for this PR, as it is a breaking change. 

Can somebody remove the `R0-silent` label please?

**Review by liamaharon — APPROVED** [review]

**Review by kianenigma — APPROVED** [review]

**Comment by muraca** [comment]
The merge failed for link errors that are not dependant on this PR. 
Can we try again?

File: prdoc/pr_3374.prdoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removes all the `pallet::getter` usages from `pallet-timestamp`, and updates depdendant runtimes accordingly.
+      The syntax `StorageItem::<T, I>::get()` should be used instead.
+
+crates: 
+  - name: pallet-timestamp
+  - name: pallet-election-provider-multi-phase
```
Comment (xlc) [inline prdoc/pr_3374.prdoc:14]: why anyone using `pallet-election-provider-multi-phas` needs to care about this PR?

Comment (tesol2y090) [reply prdoc/pr_3374.prdoc:14]: In their [test-staking-e2e](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/election-provider-multi-phase/test-staking-e2e/src/lib.rs#L49), they call the timestamp pallet.

Comment (bkchr) [reply prdoc/pr_3374.prdoc:14]: The test crate is not released.

Comment (bkchr) [inline prdoc/pr_3374.prdoc:14]:
```suggestion
```

**Review by bkchr — APPROVED** [review]

**Pull request merged**